easy-ui-react/src/Button/Button.module.scss

```diff
@@ -0,0 +1,321 @@
+@import "../styles/common";
+
+.Button {
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  border-radius: var(--ezui-shape-border-radius-base);
+  cursor: pointer;
+
+  &:focus {
+    box-shadow: var(--ezui-shadow-button);
+    outline: none;
+  }
+
+  &:disabled {
+    pointer-events: none;
+  }
+}
+
+.colorPrimary {
```

Comment:
addressed feedback for styles, let me know if i misunderstood the suggestion

Reply:
yep you understood the suggestion. it does lead to a lot of CSS, a bit more than i realized it would. i think we can clean this up a bit with component-level CSS vars like we discussed in Slack